docs/scripts/util/getStandardHTMLAttributes.ts

```diff
@@ -0,0 +1,159 @@
+import type { ComponentName } from '../types/catalog';
+
+const baseURL = `https://developer.mozilla.org/en-US/docs/Web/HTML/Element`;
+
+type ComponentNames = Exclude<
+  ComponentName,
+  | 'ButtonGroup'
+  | 'ExpanderItem'
+  | 'FieldGroupIcon'
+  | 'FieldGroupIconButton'
+  | 'MenuButton'
+  | 'MenuItem'
+  | 'Radio'
+  | 'TabItem'
+  | 'ToggleButtonGroup'
+  | 'TableBody'
+  | 'TableCell'
+  | 'TableFoot'
+  | 'TableHead'
+  | 'TableRow'
+>;
+
+type StandardHTMLAttributes = {
+  [key in ComponentNames]: {
+    element: string;
+    url?: string;
+  };
+};
+
+const standardHTMLAttributes: StandardHTMLAttributes = {
```

Comment:
Perhaps, although I kind of prefer to have all this Standard HTML Attribute info in one file instead of spread across the frontmatter for each primitive.

Reply:
I'm thinking, in the future, If we need to add a new component page, we can follow a component page template, but if we have to add it at different places, it would be harder to not miss anything 🤔

Reply:
That's a good point. I'll refactor this to put the info in the each component's frontmatter